src/parcsr_ls/par_coarse_parms.c

```diff
@@ -46,44 +35,40 @@
   marker array for coarse points
   @param coarse_dof_func_ptr [OUT]
   pointer to array which contains the function numbers for local coarse points
-  @param coarse_pnts_global_ptr [OUT]
+  @param coarse_pnts_global [OUT]
   pointer to array which contains the number of the first coarse point on each  processor and the total number of coarse points in its last element
 
   @see */
 /*--------------------------------------------------------------------------*/
 
 HYPRE_Int
-hypre_BoomerAMGCoarseParmsHost(MPI_Comm          comm,
-                               HYPRE_Int         local_num_variables,
-                               HYPRE_Int         num_functions,
-                               hypre_IntArray   *dof_func,
-                               hypre_IntArray   *CF_marker,
-                               hypre_IntArray  **coarse_dof_func_ptr,
-                               HYPRE_BigInt    **coarse_pnts_global_ptr) 
+hypre_BoomerAMGCoarseParmsHost(MPI_Comm         comm,
+                               HYPRE_Int        local_num_variables,
+                               HYPRE_Int        num_functions,
+                               hypre_IntArray  *dof_func,
+                               hypre_IntArray  *CF_marker,
+                               hypre_IntArray **coarse_dof_func_ptr,
+                               HYPRE_BigInt    *coarse_pnts_global)
```

Comment:
Since this technically changes the API, albeit the internal one, we should be extra careful with regard to breaking MFEM and PETSc here.

Reply:
Yes! I see this function is used by MFEM but not by PETSc.

Reply:
Thanks @victorapm !  Note that there are other API changes; I only pointed out the one.  We should do a more extensive test with the MFEM/PETSc codes.

Reply:
Good point. In mfem, I see `HYPRE_BigInt *cpts_global; hypre_BoomerAMGCoarseParms...  mfem_hypre_TFree(cpts_global);` The fix is simple:  `HYPRE_BigInt cpts_global[2]`. I am not sure how to deal with the version compatibility between hypre and mfem. Maybe @victorapm can help.

Reply:
Yes, I can prepare a PR for MFEM after we merge this one. The other API changes are for `hypre_BoomerAMGCoarseParmsHost` and `hypre_BoomerAMGCoarseParmsDevice` which aren't used directly by MFEM, but I can run both PETSc and MFEM tests to double check.

Reply:
Thanks for offering to help with this @victorapm!  I would recommend connecting with Tzanio before we merge.  He may prefer to start a PR based on this branch, for example.

Reply:
Ok, @rfalgout! Will do that. Thanks!